src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -55,12 +55,11 @@ trait EntityPage extends HtmlPage {
       HtmlTags.Link(href = relativeLinkTo(List("index.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet"),
     HtmlTags.Link(href = relativeLinkTo(List("template.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet"),
     HtmlTags.Link(href = relativeLinkTo(List("diagrams.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet", id = "diagrams-css"),
-    libScript("jquery.js"),
+    extScript("https://cdn.jsdelivr.net/npm/jquery@3.4.1/dist/jquery.min.js"),
```

Comment:
Self-hosted jquery.js is replaced with CDN-hosted one, because I think it can reduce size of `scaladoc` and cost of hosting, and could be faster than self-hosted. 

One caveat might be it does not allow offline view of scaladoc.
However, as a Scala user, it is unlikely to view scaladocs offline nowadays.
And also, there are already usage of CDN-hosted javascripts (`d3`-related) in scaladoc.

Reply:
Maybe a [webjar](https://www.webjars.org/) could be used instead? Then the version of jquery could be part of the build configuration and could be updated more easily?

Reply:
Do you really need offline view ?

I have no idea how to add resources from webjar to scaladoc.
Guidance is highly appreciated.

Reply:
I figured out how to use webjars in scaladoc projesct.
Will update soon.

Reply:
It's not really a problem for me personally to be honest, but I know a lot of people use tools like https://kapeli.com/dash to save docs for offline use so they can do work on trains / planes / other places with bad or no WiFi

Reply:
Ah, offline doc viewer.
That make sense for me.